.github/workflows/purge-test-resources.yaml

```diff
@@ -40,8 +41,8 @@ jobs:
           done
 
       - name: Create GitHub issue on failure
-        if: ${{ failure() }}
+        if: failure() && github.event_name == 'schedule'
```

Comment:
Does this workflow run anywhere outside of scheduled runs?

Reply:
I add this to all workflows because I don't want issues to be created during pull requests. Any suggestions? @kachawla

Reply:
What I'm trying understand is, does this workflow run for PRs? :)

Reply:
I just double-checked and, no, it does not run for PRs. My bad. I will remove this now.